addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -165,6 +165,24 @@ var SnippetEditor = Widget.extend({
         // a flickering when not needed.
         this.$target.on('transitionend.snippet_editor, animationend.snippet_editor', this.postAnimationCover);
 
+        // TODO: remove in master, handle the same in respective template.
+        const overlayButtonsTooltips = {
+            "div.o_front_back.o_send_back": _t("Send back"),
+            "div.o_front_back.o_bring_front": _t("Bring forward"),
+            "div.o_move_handle.fa-arrows": _t("Drag and move"),
+            "button.oe_snippet_remove.fa-trash": _t("Delete"),
+        };
+
+        for (const [selector, tooltip] of Object.entries(overlayButtonsTooltips)) {
+            const buttonEl = this.el.querySelector(selector);
+            if (buttonEl) {
+                Object.assign(buttonEl.dataset, {
+                    tooltip,
+                    tooltipPosition: "top"
```

Comment:
Look at that, very strange:
- It looks like the popover is on the icon instead of on the button.
- The tooltip appears and disappears continuously when you hover over the edge of the button.

https://github.com/user-attachments/assets/78920180-d3fe-4f9f-8fc0-f31eea49ae11